src/main/java/Duke.java

```diff
@@ -15,41 +26,31 @@ public static void initiliase(){
         System.out.println("Alright " +nameOfUser+ " , What can I do for you?");
         printLines();
     }
-    public static void commands(Task tasks) {
+
+    public static void readInput(Task[] tasks) {
         Scanner inputScanner= new Scanner(System.in);
         String command;
         command=inputScanner.nextLine();
```

Comment:
You might consider leaving a space before and after the "=" to improve the readability.

Reply:
Will take note of that.